zappa_settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dev": {
+        "app_function": "server.app",
+        "profile_name": "crisis-deploy",
+        "project_name": "crisis",
+        "runtime": "python3.8",
```

Comment:
Does this mean we have to use 3.8?

Reply:
It dictates the runtime used by AWS Lambda. I picked it because it was the Python version on my WSL, but other Py3 versions should still work for development.